docs/fr/docs/alternatives.md

```diff
@@ -0,0 +1,454 @@
+
+Mais ensuite, le projet a changé d'orientation.
+
+Il ne s'agissait plus d'un framework web API, le créateur devant se concentrer sur Starlette.
+
+Maintenant, APIStar est un ensemble d'outils pour valider les spécifications OpenAPI, et non un framework web.
+
+!!! info
+    APIStar a été créé par Tom Christie. Le même gars qui a créé :
+
+    * Django REST Framework
+    * Starlette (sur lequel **FastAPI** est basé)
+    * Uvicorn (utilisé par Starlette et **FastAPI**)
+
+!!! check "Inspired **FastAPI** to"
+    Exister.
+
+    L'idée de déclarer plusieurs choses (validation des données, sérialisation et documentation) avec les mêmes types Python, tout en offrant un excellent support pour les éditeurs, était pour moi une idée brillante.
+    
+    Et après avoir longtemps cherché un framework similaire et testé de nombreuses alternatives, APIStar était la meilleure option disponible.
+
+    Puis APIStar a cessé d'exister en tant que serveur et Starlette a été créé, et a constitué une meilleure base pour un tel système. Ce fut l'inspiration finale pour construire **FastAPI**.
+
+    Je considère **FastAPI** comme un "successeur spirituel" d'APIStar, tout en améliorant et en augmentant les 
+fonctionnalités, le système de typage et d'autres parties, sur la base des enseignements tirés de tous ces outils 
+précédents.
+
+## Utilisé par **FastAPI**
+
+### <a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a>
+
+Pydantic est une bibliothèque permettant de définir la validation, la sérialisation et la documentation des données (à l'aide de JSON Schema) en se basant sur les indices de type Python.
+
+Cela le rend extrêmement intuitif.
+
+Il est comparable à Marshmallow. Bien qu'il soit plus rapide que Marshmallow dans les benchmarks. Et comme il est 
+basé sur les mêmes type hints Python, le support de l'éditeur est grand.
+
+!!! check "**FastAPI** l'utilise pour"
+    Gérer toute la validation des données, leur sérialisation et la documentation automatique du modèle (basée sur le schéma JSON).
+
+    **FastAPI** prend ensuite ces données JSON Schema et les place dans OpenAPI, en plus de toutes les autres choses qu'il fait.
+
+### <a href="https://www.starlette.io/" class="external-link" target="_blank">Starlette</a>
+
+Starlette est un léger framework/toolkit <abbr title="The new standard for building asynchronous Python web">ASGI</abbr>, qui est idéal pour construire des services asyncio performants.
+
+Il est très simple et intuitif. Il est conçu pour être facilement extensible et avoir des composants modulaires.
+
+Il offre :
+
+* Des performances vraiment impressionnantes.
+* Le support des WebSockets.
+* Le support de GraphQL.
+* Les tâches d'arrière-plan en cours de traitement.
```

Comment:
```suggestion
* Les tâches d'arrière-plan
```
In my translation of features.md I had the same sentence to translate and I just said "Tâche d'arrière plan" as I had no idea how to translate "background task" in this context.